internal/distro/rhel9/images.go

```diff
@@ -317,9 +317,10 @@ func edgeRawImage(workload workload.Workload,
 	img.Users = users.UsersFromBP(customizations.GetUsers())
 	img.Groups = users.GroupsFromBP(customizations.GetGroups())
 
-	img.KernelOptionsAppend = []string{"modprobe.blacklist=vc4"}
+	img.KernelOptionsAppend = []string{"modprobe.blacklist=vc4", "rw"}
```

Comment:
Shouldn't `"ro"` be appended instead? 🤔 I'm probably missing something... Maybe a comment would be nice.

Reply:
The comment is still missing above this line...